todo.py

```diff
@@ -82,15 +82,15 @@ def list_tasks_numeric(self):
                 else:
                     print(f'{idx}. {task_name} [Priority: {priority}]')
 
-
+    #Good job with implementing this function and listing the list in alphabetical order
     def list_tasks_alphabetic(self):
         """ists all tasks in the to-do list in alphabetical order, including due dates if available."""
         if not self.tasks:
             print("No tasks in the list!")
         else:
-            sorted_tasks = []
+            sorted_tasks = [] #Although I would review the for loops to make code smaller and faster
```

Comment:
Hello! Please change the comment into a new line instead of appending it to the end of the code